catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SpatialCommand.java

```diff
@@ -62,27 +48,27 @@ protected Object executeWithSubject() throws Exception {
 
         switch (SpatialOperations.valueOf(operation.toUpperCase())) {
         case RADIUS:
-            doRadiusQuery(console, catalogProvider);
+            doRadiusQuery();
```

Comment:
@kcwire closed the todo ticket listed on line 25, but those void methods below look odd